app-emulation/aranym/aranym-1.0.2.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="Atari Running on Any Machine"
+DESCRIPTION="
+ARAnyM (Atari Running on Any Machine) is a multiplatform virtual machine for \
+running Atari ST/TT/Falcon operating systems and applications.
```

Comment:
Suggestion:
```diff
-SUMMARY="Atari Running on Any Machine"
+SUMMARY="A virtual machine for running Atari applications"
-DESCRIPTION="
-ARAnyM (Atari Running on Any Machine) is a multiplatform virtual machine for \
-running Atari ST/TT/Falcon operating systems and applications.
-"
+DESCRIPTION="ARAnyM (Atari Running on Any Machine) is a multiplatform virtual \
+machine for running Atari ST/TT/Falcon operating systems and applications."
```
(Same suggestions for `aranym-999~git.recipe`.)

Reply:
After all, you can keep the `SUMMARY="Atari Running on Any Machine"` because it sounds better.